builtin/mainmenu/tab_online.lua

```diff
@@ -111,13 +113,44 @@ local function get_formspec(tabview, name, tabdata)
 		if gamedata.fav then
 			retval = retval .. "tooltip[btn_delete_favorite;" .. fgettext("Remove favorite") .. "]"
 			retval = retval .. "style[btn_delete_favorite;padding=6]"
-			retval = retval .. "image_button[5,1.3;0.5,0.5;" .. core.formspec_escape(defaulttexturedir ..
+			retval = retval .. "image_button[4.5,1.3;0.5,0.5;" .. core.formspec_escape(defaulttexturedir ..
 				"server_favorite_delete.png") .. ";btn_delete_favorite;]"
 		end
 		if gamedata.serverdescription then
 			retval = retval .. "textarea[0.25,1.85;5.25,2.7;;;" ..
 				core.formspec_escape(gamedata.serverdescription) .. "]"
 		end
+
+		local function shortenTable(clients_list)
+			local short_clients_list = {}
+			for i = 1, math.min(5, #clients_list) do
+				short_clients_list[i] = clients_list[i]
+			end
+			return short_clients_list
+		end
+
+		local server = tabdata.lookup[tabdata.selected]
+
+		if server and server.clients_list then
+			table.sort(server.clients_list, function(a, b)
+				return string.lower(a) < string.lower(b)
+			end)
+			local clients_string = table.concat((server.clients_list), "\n")
+			local clients_string_short = table.concat(shortenTable(server.clients_list), "\n")
+			if #server.clients_list >= 5 then
+				retval = retval .. "tooltip[btn_print_clients;" .. fgettext("Clients:\n$1", clients_string_short) .. "\n..." .. "]"
+			else
+				retval = retval .. "tooltip[btn_print_clients;" .. fgettext("Clients:\n$1", clients_string) .. "]"
+			end
+			retval = retval .. "style[btn_print_clients;padding=6]"
+			retval = retval .. "image_button[5,1.3;0.5,0.5;" .. core.formspec_escape(defaulttexturedir ..
+				"server_view_clients.png") .. ";btn_print_clients;]"
+		else
+			retval = retval .. "tooltip[btn_no_clients;" .. fgettext("Clients not available.") .. "]"
+			retval = retval .. "style[btn_no_clients;padding=6]"
+			retval = retval .. "image_button[5,1.3;0.5,0.5;" .. core.formspec_escape(defaulttexturedir ..
```

Comment:
I think this should probably be just an image. An image button wrongly suggests that you can click on it and something will happen.

Reply:
I disagree. My intention was to keep the design with the "remove favorite" button consistent, in my opinion it would look off as just an image next to that button.